hardware-schema.json

```diff
@@ -141,7 +143,10 @@
         "enabled": { "$ref": "#/definitions/enabled" },
         "i2c":  { "$ref": "#/definitions/i2c" },
         "interrupt": {
-          "gpio":  { "$ref": "#/definitions/gpio" }
```

Comment:
good catch!

Reply:
Sorry, didn't unerstand it.....Can you explain me?

Reply:
In JSON the gpio node is a child node of interrupt: `interrupt/gpio`
The corresponding schema definition was wrong, the object definition was missing which defines that that gpio object has property objects.